gamestonk_terminal/economy/alphavantage_view.py

```diff
@@ -61,15 +63,17 @@ def realtime_performance_sector(raw: bool, export: str):
         df_sectors,
     )
 
-    if not raw:
-        if gtff.USE_ION:
-            plt.ion()
-        plt.show()
+    if not external_axes:
```

Comment:
this conditional doesn't work if we actually have external axes. in the other branches i am changing it to `is external_axes is None:`. Like this it works

Reply:
also in most of the places the `export_data` call is the last thing that happens in the function, so if you don't mind moving this block before the export_data call that would be great (consistency)